### ISSUE TITLE
arrow-spasing rule warns in multiline case

### ISSUE THREAD

**Issue by stremlenye** [eslint/eslint#3079]
version: `v1.0.0-rc-1`
message: `Unexpected space after =>     arrow-spacing`
In case:
```js
const test = (argument) =>
  ({ argument })
```
and configuration:
```json
"arrow-spacing": [ 2, { "before": true, "after": true }],
```
warning appears.

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by killercup** [comment]
I'm also having issues with this. [The documentation](http://eslint.org/docs/1.0.0/rules/arrow-spacing) says:

> true means there should have one space and false means no space.

Thus, I would very much like to see another option for `after` that allows a single space or a line break.

**Comment by mysticatea** [comment]
The behavior which enforces *one space* is different from other spacing rules. It's a work of `no-multi-spaces`.
So I guess this rule should allow multi-spaces and line-ending (together with spaces of indent).
